auto_round/script/llm.py

```diff
@@ -588,8 +588,8 @@ def tune(args):
             if args.eval_task_by_task:
                 eval_task_by_task(user_model, device=device_str, tasks=args.tasks, batch_size=args.eval_bs)
             else:
-                if args.eval_bs is None:
-                    args.eval_bs = "auto"
+                if args.eval_bs is None or args.eval_bs == "auto":
+                    args.eval_bs = 16
```

Comment:
better add a logger warning, "This API does not support auto currently, reset `eval_bs` to 16 "